uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.js

```diff
@@ -19,37 +19,28 @@ define([
 
       var defaultDocumentStatus = ['1', '2']; // 1: 'awaiting upload' | 2: 'awaiting approval
```

Comment:
Well actually, `['1', '2']` are the status ids. So i guess it is more preferable to rename it as `deafaultStatusIds`. What do you say?